### ISSUE TITLE
Label for Notifications button reads "t('notifications', 'Notifications')"

### ISSUE THREAD

**Issue by MarcoZehe** [nextcloud/server#20121]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.

### Steps to reproduce
1. Open the NextCloud front page and log in.
2. Inspect the accessible name for the Notifications button with your favorite accessibility inspection tool.

### Expected behaviour
The name should be "Notifications", possibly followed by a number of unread notifications.

### Actual behaviour
The name reads `t('notifications', 'Notifications')`, looking like a malformed call to the localization mechanism.

NextCloud 17.1.

CC @nextcloud/accessibility

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/notifications/blob/ae40719383f66fab7af799e1a4516aa48fbec607/src/App.vue#L8
Moving to the notifications repo! Thanks you so much for digging this @MarcoZehe !
